### PR TITLE
Update NOTICE

### PR DESCRIPTION
**Pull request by rgreenberg1**
Name changes, removed DS Engine.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -9,9 +9,9 @@ Version 2.0, an Apache-compatible license.
 * For a copy of the Apache License Version 2.0, please see LICENSE
   as included in the top-level directory of the deepsparse GitHub repository.
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the Neural Magic Engine License (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC). Example files and scripts included in the deepsparse GitHub repository are licensed under the Apache License Version 2.0 (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+DeepSparse Community is licensed under the Neural Magic DeepSparse License (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC). Example files and scripts included in the deepsparse GitHub repository are licensed under the Apache License Version 2.0 (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```
Comment (jeanniefinks) [inline NOTICE:12]:
```suggestion
DeepSparse Community is licensed under the Neural Magic DeepSparse License (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC). Some source code, example files and scripts included in the deepsparse GitHub repository or directory are licensed under the Apache License Version 2.0 (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
See minor edits so we're consistent in other places using the same language and it can be used across Docs, GH, and PyPI.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ limitations under the License.
     <img alt="tool icon" src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />
     &nbsp;&nbsp;DeepSparse Engine
   </h1>
-  <h3> Sparsity-aware neural network inference engine for GPU-class performance on CPUs </h3>
+  <h3> An inference runtime offering GPU-class performance on CPUs and APIs to integrate ML into your application. </h3>
```
Comment (jeanniefinks) [inline README.md:23]:
```suggestion
  <h3> An inference runtime offering GPU-class performance on CPUs and APIs to integrate ML into your application.</h3>
```

Comment (jeanniefinks) [reply README.md:23]: taglines/subheadlines  don't normally get periods at the end as we haven't been doing that stylistically throughout the site.

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
tiny edit!

**Comment by rgreenberg1** [comment]
Updating the Notice file in DeepSparse and README.

File: README.md [hunk 3]
```diff
@@ -266,10 +266,10 @@ For more general questions about Neural Magic, complete this [form.](http://neur
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC) 
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic DeepSparse Community License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```
Comment (jeanniefinks) [inline README.md:269]:
```suggestion
[DeepSparse Community](https://docs.neuralmagic.com/products/deepsparse) is licensed under the [Neural Magic DeepSparse Community License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

Comment (rgreenberg1) [reply README.md:269]: This is the right one, yes?

**Review by jeanniefinks — APPROVED** [review]

**Review by bnellnm — APPROVED** [review]

**Pull request merged**